src/fhir_mcp_server/server.py

```
@@ -283,42 +298,48 @@ async def search(
         except ValueError as ex:
             logger.exception(
                 f"User does not have permission to perform FHIR '{type}' resource search operation. Caused by, ",
                 exc_info=ex,
             )
-            return await get_operation_outcome(
-                code="forbidden",
-                diagnostics=f"The user does not have the rights to perform search operation.",
+            return format_response(
+                await get_operation_outcome(
+                    code="forbidden",
+                    diagnostics=f"The user does not have the rights to perform search operation.",
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)